requirements.txt

```diff
@@ -1,2 +1,2 @@
-requests~=2.31.0
+requests~=2.32.2
 oauthlib~=3.2.2
```

Comment:
Please use `requests>=2.32.2` instead of `requests~=2.32.2` (or requests~=2.32.0)?

Same for `oauthlib>=3.2.2`.